sgxs-loaders/tests/test_linux_drivers.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash -e
+#
+# Prior to running this script, make sure the system has the following
+# configuration:
+#
+# * Linux: 4.15 generic kernel (e.g. not the Azure kernel)
+# * Test dependencies: build-essential dkms docker.io
+# * EDP dependencies: pkg-config libssl-dev protobuf-compiler
+# * Rust: latest nightly installed with x86_64-fortanix-unknown-sgx target
+# * No other SGX drivers or software installed (from any vendor)
+#
+# Note: you can't run this script multiple times because the FLC drivers make
+# CPU configuration changes that the non-FLC drivers don't support.
```

Comment:
How do you recover from running this test once? Is rebooting the only option?

Reply:
Rebooting is easiest. You could manually restore the PUBKEYHASH MSRs but I didn't invest time in figuring out how.